Apps/WebClient/src/ClientApp/src/components/report/covid19.vue

```diff
@@ -92,13 +86,29 @@ export default class COVID19ReportComponent extends Vue {
         });
     }
 
+    @Watch("isLaboratoryLoading")
+    @Emit()
+    private onIsLoadingChanged() {
+        return this.isLaboratoryLoading;
+    }
+
+    @Watch("isEmpty")
+    @Emit()
+    private onIsEmptyChanged() {
+        return this.isEmpty;
+    }
+
     private created() {
         this.logger = container.get<ILogger>(SERVICE_IDENTIFIER.Logger);
         this.retrieveLaboratory({ hdid: this.user.hdid }).catch((err) => {
             this.logger.error(`Error loading Covid19 data: ${err}`);
         });
     }
 
+    private mounted() {
+        this.$emit("on-is-empty-changed", this.isEmpty);
```

Comment:
I don't understand the need of this, is it the equivalent to this.onIsEmptyChanged()? if so I'd rather have intelliSense than depending on the method key as a string.

Reply:
Good call. Updated it to reference the method instead of a fresh call